tdrs-backend/tdpservice/parsers/schema_defs/tanf/t2.py

```diff
@@ -11,145 +11,376 @@
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_BLACK', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_HAWAIIAN', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='RACE_WHITE', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='MARITAL_STATUS', result_function=validators.oneOf((1, 2, 3, 4, 5)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                      result_field='PARENT_WITH_MINOR_CHILD', result_function=validators.oneOf((1, 2, 3)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='EDUCATION_LEVEL', result_function=validators.oneOf((1, 2, 3, 4, 5,
+                                                                                        6, 7, 8, 9, 10,
+                                                                                        11, 12, 13, 14,
+                                                                                        15, 16, 98, 99)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='CITIZENSHIP_STATUS', result_function=validators.oneOf((1, 2)),
+                ),
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2, 3)),
+                      result_field='COOPERATION_CHILD_SUPPORT', result_function=validators.oneOf((1, 2, 9)),
+                ),
+            # TODO THIS ISNT EXACTLY RIGHT SINCE FED TIME LIMIT IS A STRING.
+            validators.if_then_validator(
+                      condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                      result_field='MONTHS_FED_TIME_LIMIT', result_function=validators.isLargerThan(1),
+                ),
+            # TODO THIS ISNT EXACTLY RIGHT SINCE FED TIME LIMIT IS A STRING.
```

Comment:
todo still relevant? the schema has `MONTHS_FED_TIME_LIMIT` as `number`

Reply:
I had changed this field to number, but missed the TODO